script/source-setup/pip

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -e
+
+if [ -z "$(which pip)" ]; then
+  echo "A local pip installation is required for python development." >&2
+  exit 127
+fi
+
+if [ -z "$(which virtualenv)" ]; then
+    echo "A local virtualenv installation is required for python development." >&2
+    exit 127
+fi
+
+
+# setup test fixtures
```

Comment:
can you add support for a "-f" flag that cleans up any existing fixtures before doing the setup steps?  e.g. something like

https://github.com/github/licensed/blob/d22011c9dd162593919752410a642696c9ade454/script/source-setup/go#L14-L16

Reply:
👍